pkg/agent/controller/noderoute/node_route_controller.go

```diff
@@ -52,6 +52,9 @@ const (
 	ovsExternalIDNodeName = "node-name"
 
 	nodeRouteInfoPodCIDRIndexName = "podCIDR"
+
+	IPv4BitLen = 32
```

Comment:
there are constants for these lengths in net package: https://golang.org/pkg/net/

Reply:
Thanks, I only found bytes length constants, so I changed the code to:
```
IPv4BitLen = net.IPv4len * 8
IPv6BitLen = net.IPv6len * 8
```

Reply:
Thanks for checking. 
Realized that this is available here: https://github.com/vmware-tanzu/antrea/blob/main/pkg/util/ip/ip.go#L26

Reply:
Thanks, I capitalized these constant and updated the code.